src/plugins/util.commerce.ts

```diff
@@ -177,16 +176,13 @@ export const getNormalPrice = (item: Node): Node | undefined => {
       if (variants.type !== Type.ARRAY || size === 0) {
         return DEFAULT_MONEY_NODE;
       }
-      let moneyNode = variants.get(0);
+      let moneyNode = variants.get(0).path(['priceMoney']);
```

Comment:
This was an existing bug that I found from product-price tests failing. The corresponding line in template-compiler is [here](https://github.com/Squarespace/template-compiler/blob/main/core/src/main/java/com/squarespace/template/plugins/platform/CommerceUtils.java/#L214-L215)